lib/integration/dynamodbConnector.ts

```diff
@@ -1,15 +1,21 @@
 import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
 import { DynamoDBDocumentClient } from "@aws-sdk/lib-dynamodb";
 
+let dynamodbInstance: DynamoDBDocumentClient | undefined = undefined;
+
 /**
  * Helper function to instantiate DynamoDB and Document client.
  * https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/dynamodb-example-document-client.html
```

Comment:
since we are now using v3, we could do the new way

Reply:
Sure I can try now. I was thinking about testing this fix and then refactor the code so that all AWS services are going through such type of singleton.

Let me rework this one first and then we can test before modifying all the others.